### PR TITLE
Feature/reset depth

### PR DESCRIPTION
**Pull request by leoTiez**

### CODE REVIEW

File: RCNN_tf.py [hunk 1]
```diff
@@ -542,7 +545,7 @@ def train(self, train_data_feats, train_data_labels, val_data_feats,
                 sess.run(self.train_init_op, feed_dict={
                     self.input_placeholder: train_data_feats,
                     self.output_placeholder: train_data_labels,
-                    self.num_data_placeholder: batch_size
+                    self.num_data_placeholder: batch_size,
```
Comment (bt-s) [inline RCNN_tf.py]: Comma could be removed :)

File: RCNN_tf.py [hunk 2]
```diff
@@ -474,6 +476,7 @@ def __init__(
 
     def train(self, train_data_feats, train_data_labels, val_data_feats,
               test_data_feats, test_data_labels, val_data_labels,
+              learning_depth=3, test_depth=6,
```
Comment (bt-s) [inline RCNN_tf.py]: Personally, I would find ```training_depth``` a more apt naming than ```learning_depth```.

File: RCNN_tf.py [hunk 3]
```diff
@@ -474,6 +476,7 @@ def __init__(
...
               batch_size=100, epochs=7, dropout_rate=0.2,
               adaptive_learning_factor=.1, performance_update_threshold=3,
               min_difference_accuracy=.005, create_graph=True, print_vars=True,
```
Comment (bt-s) [inline RCNN_tf.py:482]: Please describe ```learning_depth``` and ```test_depth``` in the docstring, to keep it consistent and complete.

File: main.py [hunk 4]
```diff
@@ -200,9 +201,14 @@ def main(argv):
...
 
     if not do_grid_search:
+        adaptive_learning_factor = 1
+        learning_depth_ = 3
+        test_depth_ = 3
         if train_args:
-            if train_args.depth is not None:
-                recurrent_depth_ = train_args.depth
+            if train_args.depth_learning is not None:
+                learning_depth_ = train_args.depth_learning
```
Comment (bt-s) [inline main.py]: Maybe change ```depth_learning``` to ```learning_depth```.
Idem for ```depth_test``` to ```test_depth```.

**Review by bt-s — APPROVED** [review]
See comments.

File: RCNN_tf.py (hunk 1 above)
Comment (leoTiez) [reply RCNN_tf.py]: Good point

File: main.py (hunk 4 above)
Comment (leoTiez) [reply main.py]: Will do

File: RCNN_tf.py (hunk 2 above)
Comment (leoTiez) [reply RCNN_tf.py]: Can do, but it makes it easier for the auto completion. But if you feel happy with it

File: RCNN_tf.py (hunk 3 above)
Comment (leoTiez) [reply RCNN_tf.py:482]: :+1:

**Comment by leoTiez** [comment]
Changed the commented passages, will merge now

**Pull request merged**